pkg/cluster/template/config/prometheus.go

```diff
@@ -185,6 +198,19 @@ func (c *PrometheusConfig) AddDMWorker(ip string, port uint64) *PrometheusConfig
 	return c
 }
 
+// AddRemote add remote read/write server config for prometheus
+func (c *PrometheusConfig) AddRemote(remoteRead *meta.RemoteReadServerConfig, remoteWrite *meta.RemoteWriteServerConfig) *PrometheusConfig {
```

Comment:
In our previous implement, `AddXXX` will add a new configuration to the existing slice, IMO, you can rename it to `SetRemote`, one more thing, I think you can split this function into `SetRemoteWrite` and `SetRemoteRead`

Reply:
As I've suggested previously, don't define the struct in the module `meta`, so I think maybe you should pass the `RemoteWriteURL`, `Capacity` ... directly

Reply:
use SetRemoteConfig instead